src/js/components/Select/__tests__/SelectUncontrolled-test.js

```diff
@@ -28,39 +41,40 @@ describe('Select', () => {
     expect(component.toJSON()).toMatchSnapshot();
```

Comment:
we can move this line to the first test of axe, and get rid of this 'a11yTitle' test since its testing are already included in the axe testing.

Reply:
Has been changed in the most recent push